### DISCUSSION TITLE
Get hessian of Lagrangian in Optistack?

### DISCUSSION THREAD

**Discussion by brunomorampc** [casadi/casadi#3330] [Q&A] [answered]
How can I get the values of the Hessian of the Lagrangian with Optistack for every iteration?
I found this https://groups.google.com/g/casadi-users/c/awTvuvT33B4/m/WNZ94N4hCAAJ but it is not clear to me where I can find the solver instance in the opti object.

**Comment by jgillis** [comment, marked as answer]
`opti.debug.casadi_solver` iirc

**Reply by sixtosevenf** [reply]
On my side, the command `opti.debug.casadi_solver ` is not working with the following error:

> >> opti.debug.casadi_solver
> 
> Unrecognized method, property, or field 'casadi_solver' for class 'casadi.OptiAdvanced'.
